### PR TITLE
gzexe: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

This command has only one option, so the tldr page is rather short.

### CODE REVIEW

File: pages/linux/gzexe.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+> A compressor that compresses executable files while keeping them executable.
+> More information: <https://manned.org/gzexe.1>.
+
+- Compress an executable file in-place:
+
+`gzexe {{path/to/executable}}`
+
+- Decompress a compressed executable in-place:
+
+`gzexe -d {{path/to/compressed/executable}}`
```
Comment (kbdharun) [inline pages/linux/gzexe.md:12]:
```suggestion
`gzexe -d {{path/to/compressed_executable}}`
```

File: pages/linux/gzexe.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# gzexe
+
+> A compressor that compresses executable files while keeping them executable.
```
Comment (kbdharun) [inline pages/linux/gzexe.md:3]:
```suggestion
> Compresses executable files while keeping them executable.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/gzexe.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# gzexe
+
+> Compresses executable files while keeping them executable.
```
Comment (acuteenvy) [inline pages/linux/gzexe.md:3]:
```suggestion
> Compresses executable files while keeping them executable.
> Back up the original file, appending `~` to its name and create a shell script that uncompresses and executes the binary inside it.
```

File: pages/linux/gzexe.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://manned.org/gzexe.1>.
+
+- Compress an executable file in-place:
+
+`gzexe {{path/to/executable}}`
+
+- Decompress a compressed executable in-place:
```
Comment (acuteenvy) [inline pages/linux/gzexe.md:10]:
```suggestion
- Decompress a compressed executable in-place (i.e. convert the shell script back to an uncompressed binary):
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**